experimental/cmd/dive/clipboard.go

```
@@ -0,0 +1,214 @@
+			_ = runClipboardTool(path, []string{"--primary"}, text)
+		}
+	case os.Getenv("DISPLAY") != "":
+		if path, err := exec.LookPath("xclip"); err == nil {
+			_ = runClipboardTool(path, []string{"-selection", "primary"}, text)
+		}
+	}
+}
+
+// runClipboardTool feeds text to a command's stdin and waits for it.
+func runClipboardTool(name string, args []string, text string) error {
+	cmd := exec.Command(name, args...)
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)